sklearn/utils/estimator_checks.py

```diff
@@ -2070,14 +2070,14 @@ def check_estimators_pickle(name, estimator_orig, readonly_memmap=False):
     if readonly_memmap:
         unpickled_estimator = create_memmap_backed_data(estimator)
     else:
-        # pickle and unpickle!
+        # No need to touch the file system in that case.
         pickled_estimator = pickle.dumps(estimator)
         module_name = estimator.__module__
         if module_name.startswith("sklearn.") and not (
             "test_" in module_name or module_name.endswith("_testing")
         ):
-            # strict check for sklearn estimators that are not implemented in test
-            # modules.
+            # strict check for sklearn estimators that are not implemented
+            # in test modules.
```

Comment:
```suggestion
            # strict check for sklearn estimators that are not implemented in test
            # modules.
```

Original format was ok now that de-indented this code block back to it's original level. Let's wait for the current CI run to complete before pushing this cosmetic fix.